paz/pipelines/keypoints.py

```diff
@@ -384,3 +389,31 @@ def call(self, image):
             image = self.draw(image, keypoints)
         image = self.draw_boxes(image, boxes2D)
         return self.wrap(image, boxes2D, keypoints2D, keypoints3D)
+
+
+class SimpleBaselines(pr.Processor):
+    def __init__(self, estimate_keypoints_3D, args_to_mean,
+                 h36m_to_coco_joints2D):
```

Comment:
I think I would hardcode the `h36m_to_coco_joints2D` and `args_to_mean` variables in the constructor and just since is specific to this pair of HigherHRNet and SimpleBaselines.